packages/documentation/docs/injection.md

```diff
@@ -1,4 +0,0 @@
----
-id: injection
```

Comment:
Do we need this file? What's the purpose of keeping it here?

Reply:
nope we don't need it. It's deleted